variables.tf

```
@@ -64,3 +64,9 @@ variable "user_data" {
   default     = null
   description = "Instance user data"
 }
+
+variable "affinity_group_ids" {
+  description = "Host affinity group IDs if required for the instance"
+  type        = list(string)
+  default     = []
+}
```

Variables are sorted alphabetically and the field order should be `type`, `default` then `description`:

```suggestion
variable "affinity_group_ids" {
  type        = list(string)
  default     = []
  description = "Host affinity group IDs"
}
```